### PR TITLE
kdyby/curl-ca-bundle is abandoned

### PR DESCRIPTION
**Pull request by integer**
Use composer/ca-bundle instead.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,9 +21,10 @@
 		"nette/di": "~2.3@dev",
 		"nette/http": "~2.3@dev",
 
-		"kdyby/curl-ca-bundle": "~1.0",
+		"composer/ca-bundle": "~1.0",
 		"ext-curl": "*",
-		"ext-json": "*"
+		"ext-json": "*",
+		"symfony/process": "~3.1"
```
Comment (hrach) [inline composer.json]: why are you addeing symfony/process?

Comment (integer) [inline composer.json]: Because of this suggest in composer/ca-bundle: https://github.com/composer/ca-bundle/blob/master/composer.json#L34

Comment (simPod) [inline composer.json]: docs say `but can be ignored on PHP 5.5.6+`
`nette/application` etc. already requires php 5.6. It should be removed I think, not neeeded

Comment (integer) [inline composer.json]: symfony/process dependency removed. PR updated

**Comment by fprochazka** [comment]
Thank you! I've went with #59

**Pull request closed without merging**